gradio/chat_interface.py

```diff
@@ -249,7 +249,10 @@ def __init__(
 
         with self:
             self.saved_conversations = BrowserState(
-                [], storage_key="_saved_conversations"
+                [], storage_key=f"_saved_conversations_{self._id}"
```

Comment:
Adds a `self._id` in case your Gradio app has multiple `gr.ChatInterface`-s
```suggestion
                [], storage_key=f"_saved_conversations_{self._id}"
```